release/pkg/assets/archives.go

```diff
@@ -0,0 +1,142 @@
+package assets
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+
+	assettypes "github.com/aws/eks-anywhere/release/pkg/assets/types"
+	"github.com/aws/eks-anywhere/release/pkg/filereader"
+	releasetypes "github.com/aws/eks-anywhere/release/pkg/types"
+	artifactutils "github.com/aws/eks-anywhere/release/pkg/util/artifacts"
+	sliceutils "github.com/aws/eks-anywhere/release/pkg/util/slices"
+)
+
+func getArchiveAssets(rc *releasetypes.ReleaseConfig, archive *assettypes.Archive, projectPath, gitTag, eksDReleaseChannel, eksDReleaseNumber, kubeVersion string) (*releasetypes.ArchiveArtifact, error) {
```

Comment:
How difficult would it be to add tests to this method?

Reply:
Done!